src/main/java/frc/robot/commands/RunBallShooter.java

```diff
@@ -9,37 +9,28 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.robot.subsystems.BallShooterSubsystem;
-import frc.robot.subsystems.BallFeederSubsystem;
 
 /**
  * A command to turn on the launch motor to a known desired power
  */
 public class RunBallShooter extends CommandBase
 {
    private final BallShooterSubsystem ballShooter;
-   private final BallFeederSubsystem ballFeeder;
    private final double m_motorPowerBottom;
    private final double m_motorPowerTop;
-   private final double m_feederPower;
 
-   public RunBallShooter(BallShooterSubsystem shooterSubsystem, BallFeederSubsystem feederSubsystem, double shooterBottomPower, double shooterTopPower, double feederPower)
+   public RunBallShooter(BallShooterSubsystem shooterSubsystem, double shooterBottomPower, double shooterTopPower)
    {
       ballShooter = shooterSubsystem;
-      ballFeeder = feederSubsystem;
       m_motorPowerBottom = shooterBottomPower;
       m_motorPowerTop = shooterTopPower;
-      m_feederPower = feederPower;
-      addRequirements(shooterSubsystem, feederSubsystem);
+      addRequirements(shooterSubsystem);
    }
 
    @Override
    public void initialize()
    {
       ballShooter.setMotorPower(m_motorPowerBottom, m_motorPowerTop);
-      ballFeeder.setMotorPower(m_feederPower);
-      System.out.println("Power Top: " + m_motorPowerTop);
-      System.out.println("Power Bot: " + m_motorPowerBottom);
-
```

Comment:
Oh, I love this line of code. Completely empty - so useful. I guess it serves to remind us of the endless amount of space in this vast universe. So, I suppose it's poetic? Way to go and use analogy.